docs/docs/ref-10-glossary.md

```diff
@@ -189,5 +190,7 @@ type ReactComponent<TProps> = {
   props : TProps,
   render : () => ReactElement
 };
+
+type ReactStatelessComponent<TProps> = (TProps) => ReactElement;
```

Comment:
The internal naming can sometimes differ from what we want to expose publicly (e.g. we call shared modules `isomorphic` in the source but we never use this word in the docs). So maybe this is one of those cases.

cc @sebmarkbage for terminology clarification